doc/modules/qualitymetrics/sd_ratio.rst

```diff
@@ -0,0 +1,38 @@
+Standard Deviation (SD) ratio (:code:`sd_ratio`)
+==============================================
+
+Calculation
+-----------
+
+All spikes from the same neuron should have the same shape. This means that at the peak of the spike, the standard deviation of the voltage should be the same as that of noise. If spikes from multiple neurons are grouped in a single unit, chances are the standard deviation of spikes amplitude is going to increase.
+
+This metric, first described [Pouzat]_ then adapted by Wyngaard, Llobet & Barbour (paper in writing), returns the ratio between both standard deviations:
```

Comment:
```suggestion
This metric, first described [Pouzat]_ then adapted by Wyngaard, Llobet & Barbour (in preparation), returns the ratio between both standard deviations:
```
In US academic English we typically use this to signify writing unless yours is the British way in which case it is fine to leave alone.